### PR TITLE
feat: fancy_logger_test

### PR DESCRIPTION
**Pull request by nesquikm**
## Description

Add fancy_logger tests

## Type of Change

- [x] 🧹 Code refactor
- [x] ✅ Build configuration change

### CODE REVIEW

File: packages/fancy_logger/lib/src/db_logger.dart [hunk 1]
```diff
@@ -181,6 +205,15 @@ class DbLogger extends AbstractLogger {
 
     unawaited(_database.execute(query));
   }
+
+  /// Clear logs (for debug purposes only)
+  Future<void> clearAllLogs() async {
```
Comment (Alex-A4) [inline packages/fancy_logger/lib/src/db_logger.dart:210]: You can mark it as `@visibleForTest`

File: packages/fancy_logger/test/src/writable_log_record.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// ignore_for_file: prefer_const_constructors
```
Comment (Alex-A4) [inline packages/fancy_logger/test/src/writable_log_record.dart:1]: I'm not sure, but as I remember, files with tests must contains `_test` suffix in file name

**Review by Alex-A4 — APPROVED** [review]

File: packages/fancy_logger/lib/src/db_logger.dart (hunk 1 above)
Comment (nesquikm) [reply packages/fancy_logger/lib/src/db_logger.dart:210]: Yeah, thank you! Fixed.

File: packages/fancy_logger/test/src/writable_log_record.dart (hunk 2 above)
Comment (nesquikm) [reply packages/fancy_logger/test/src/writable_log_record.dart]: [Exactly](https://dartcodemetrics.dev/docs/rules/common/prefer-correct-test-file-name), thank you!

**Pull request merged**